Cargo.toml

```diff
@@ -256,6 +256,7 @@ members = [
     "rs/registry/admin-derive",
     "rs/registry/canister",
     "rs/registry/canister/api",
+    "rs/registry/canister/chunks",
```

Comment:
This seems useful in other contexts, and I wonder if it should be in nervous_system